tests/chaos/oadp/conftest.py

```diff
@@ -0,0 +1,81 @@
+import datetime
+import logging
+
+import pytest
+from ocp_resources.datavolume import DataVolume
+from ocp_resources.utils.resource_constants import ResourceConstants
+
+from tests.chaos.oadp.utils import get_current_storage_class_config
+from utilities.constants import FILE_NAME_FOR_BACKUP, TEXT_TO_TEST, TIMEOUT_3MIN, TIMEOUT_5MIN
+from utilities.infra import ExecCommandOnPod, wait_for_node_status
+from utilities.oadp import VeleroBackup, create_rhel_vm, is_storage_class_support_volume_mode
+from utilities.storage import write_file
+
+LOGGER = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def rhel_vm_with_dv_running(request, chaos_namespace, snapshot_storage_class_name_scope_module):
+    """
+    Create a RHEL VM with a DataVolume.
+    """
+    sc_name, sc_config = get_current_storage_class_config(pytest_config=snapshot_storage_class_name_scope_module)
+
+    param = request.param
+    volume_mode = param.get("volume_mode", sc_config["volume_mode"])
+    access_mode = param.get("access_mode", sc_config["access_mode"])
+
+    if volume_mode == DataVolume.VolumeMode.BLOCK and not is_storage_class_support_volume_mode(sc_name, "Block"):
+        pytest.skip(f"Storage class {sc_name!r} doesn't support Block mode")
```

Comment:
do not create programmatic skips...

you pass `storage-class-matrix` param so you already know if cluster has Block-supported storage or not..